File: TwoSum/two_sum.py
from typing import List


def twoSum(nums: List[int], target: int) -> List[int]:
    for i in range(len(nums)):
        num = target - nums[i]
        if num in nums[i + 1:len(nums)]:

            return [i, nums[i + 1:].index(num) + 1 + i]

File: TwoSum/two_sum.go
package main

func twoSum(nums []int, target int) []int {
	for i, num := range nums {
		for j := i + 1; j < len(nums); j++ {
			if nums[j] == target-num {
				return []int{i, j}
			}
		}
	}
	panic("error")
}
